docs/docs/external/getting-started/test-network/setup-local-docker.md

```diff
@@ -22,7 +24,7 @@ Before starting, make sure you have the following software installed on your hos
 - Curl: _install using package manager, like `apt` on Debian/Ubuntu Linux_
 - Git: [sample instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - Docker: [sample instructions](https://docs.docker.com/engine/install/) (Latest version)
-- Docker-Compose: [sample instructions](https://docs.docker.com/compose/install/) (Latest version)
+- Docker-Compose: [sample instructions](https://docs.docker.com/compose/install/) (Version 1.28.2 or above)
```

Comment:
Do we have a hard dependency on this version?

Reply:
Technically, "Latest version" would apply for this too then. But it's good to specify the oldest supported version. We ought to do that for the other items in the list too at some point.